ext/duckdb/result.c

```diff
@@ -100,11 +96,7 @@ static VALUE to_ruby_obj(duckdb_result *result, idx_t col_idx, idx_t row_idx) {
         p = duckdb_value_varchar(result, col_idx, row_idx);
         if (p) {
             obj = rb_str_new2(p);
-#ifdef HAVE_DUCKDB_FREE
-            duckdb_free(p);
```

Comment:
don't remove this line

Reply:
Replaced free() with duckdb_free() in line 104.